src/inmanta/loader.py

```diff
@@ -662,3 +690,50 @@ def module_in_prefix(module: types.ModuleType, prefix: str) -> bool:
     for mod_name in loaded_modules:
         del sys.modules[mod_name]
     importlib.invalidate_caches()
+
+
+@dataclass(frozen=True)
+class PythonModule:
+    name: str
+    version: str
+    files_in_module: list["SourceInfo"]
+
+
+class SourceInfo(BaseModel):
+    """This class is used to store information related to source code information"""
```

Comment:
Could you document how this is distinct from ModuleSource and PythonModule?

Reply:
removed